catalog/meta-chatbot-inaccuracies.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Meta's AI bot trained on scientific info put out biased and inaccurate articles
+companies: ["Meta"]
+categories: ["Inaccuracies", "Harmful Bias", "Misrepresentation"]
+demonstrative: false
+modalities: ["Chatbot"]
+date: 2022-11-22
+models: []
+sources:
+  - "https://gizmodo.com/meta-ai-bot-galactica-1849813665"
+---
+
+Meta's AI bot Galactica claimed Black people "don’t have a language of their own" and that Stanford researchers are building "gaydar" software to find gay people on Facebook. In addition, the bot incorrectly cited scientists, potentially hurting their reputations.
```

Comment:
Think we could make the link stronger (accept only if it accords with your understanding of the facts though):

```suggestion
Meta's AI bot Galactica claimed Black people "don’t have a language of their own" and that Stanford researchers are building "gaydar" software to find gay people on Facebook. In addition, the bot substantiated its problematic claims with incorrect citations real scientists, potentially damaging those scientists' reputations.
```